client/my-sites/checklist/onboardingChecklist.js

```diff
@@ -5,81 +5,87 @@
  */
 import page from 'page';
 import { isDesktop } from 'lib/viewport';
+import { translate } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 const tasks = {
 	avatar_uploaded: {
-		title: 'Upload your profile picture',
-		description:
-			'Who’s the person behind the site? Personalize your posts and comments with a custom profile picture.',
-		duration: '2 mins',
-		completedTitle: 'You uploaded a profile picture',
-		completedButtonText: 'Change',
+		title: translate( 'Upload your profile picture' ),
+		description: translate(
+			'Who’s the person behind the site? Personalize your posts and comments with a custom profile picture.'
+		),
+		duration: translate( '%d min', '%d mins', { count: 2, args: [ 2 ] } ),
+		completedTitle: translate( 'You uploaded a profile picture' ),
+		completedButtonText: translate( 'Change' ),
 		url: '/me',
 		image: '/calypso/images/stats/tasks/upload-profile-picture.svg',
 		tour: 'checklistUserAvatar',
 	},
 	blogname_set: {
-		title: 'Give your site a name',
-		description: 'Give your site a descriptive name to entice visitors.',
-		duration: '1 min',
-		completedTitle: 'You updated your site title',
-		completedButtonText: 'Edit',
+		title: translate( 'Give your site a name' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Give your new site a name.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).